internal/service/cluster/cluster.go

```diff
@@ -162,6 +163,7 @@ func (s *Service) Definition() service.Definition {
 			// Cluster depends on the HTTP service to work properly.
 			http_service.ServiceName,
 		},
+		Stability: featuregate.StabilityBeta,
```

Comment:
In the PR comment I think you said this should be stable. I think clustering should be marked as stable, that's what the docs currently say.